submissions/description/sensitive_data_exposure/critically_sensitive_data/template.md

```diff
@@ -0,0 +1,30 @@
+# Critically Sensitive Data Exposure
+
+## Overview of the Vulnerability
+
+Disclosure of critically sensitive data occurs when the data is not behind an authorization barrier, placing critically sensitive data, such as secrets or internal API keys, at risk. This application discloses critical sensitive data which an attacker could use to take over the account and access, delete, or modify data from within the application, amongst other attack vectors.
```

Comment:
Soem technical wording changes The disclosure of critically sensitive data does not always involve API keys or credentials, it could be documents, or company secrets. I would recommend rewording below for something that would work better

> Disclosure of critically sensitive data occurs when the data is properly secured, allowing critically sensitive data, such as secrets, API keys, or other data critical to business operation to be exposed. The exposure of this data can be described as critically sensitive as it's exposure would likely cause a high priority incident.